package.json

```diff
@@ -136,12 +136,10 @@
     "expo-linking": "^7.0.3",
     "expo-localization": "~16.0.0",
     "expo-media-library": "~17.0.3",
-    "expo-navigation-bar": "~4.0.4",
     "expo-notifications": "~0.29.11",
     "expo-screen-orientation": "^8.0.2",
     "expo-sharing": "^13.0.0",
     "expo-splash-screen": "~0.29.18",
-    "expo-status-bar": "~2.0.0",
```

Comment:
Both `expo-navigation-bar` and `expo-status-bar` must be replace with `SystemBars` to avoid interferences (same goes for the builtin `StatusBar`).